ovos_plugin_common_play/ocp/media.py

```diff
@@ -1,5 +1,6 @@
 from typing import Optional, Tuple, List, Union
 
+from mycroft.messagebus import MessageBusClient
```

Comment:
`from mycroft_bus_client`

and in the process of moving to `ovos_bus_client` in ovos-core 0.0.8

Reply:
Do we need a new bus client package, or is @forslund an owner of mycroft_bus_client and we can push changes there? There are a lot of references and it'd be nice to not have to update all of them...

Reply:
the new session stuff and future updates need changes that wouldnt make it upstream, this is being done as part of https://github.com/OpenVoiceOS/ovos-core/pull/292

right now the main thing is the injection of session data, but this is also cleaning up those messy imports we had in core because of that "utterance_remainder" method in Message not being present in mycroft-bus-client

there is also the whole user preferences/config/lang stuff coming from message that i think makes all sense to implement under ovos-bus-client as they have nothing to do with mycroft. 

the Message object itself remains compatible everywhere via metaclass magic (with unittests), the migration should be painless and each project can do it at it's own pace

Reply:
Discussion is out of scope of this PR, resolving.